### PR TITLE
Prepare 3.0.0 Beta

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -60,21 +73,31 @@ Lib/
 Lib/slapdtest.py
 * Automatically try some common locations for SCHEMADIR
 * Ensure server is stopped when the process exits
+* Check for LDAP schema and slapd binaries
```
Comment (tiran) [inline CHANGES]: slapdtest also includes certificates (CA, server, client) and configuration for TLS client auth.

Apropos certificates, the test certs are **not** installed with ``slapdtest``. Is slapdtest shipped in standard distributions?

Comment (encukou) [reply CHANGES]: It is in the sdist, yes.

Comment (encukou) [reply CHANGES]: But actually, it's python-ldap's test suite that includes the certificates, not not `slapdtest.py`.
This should be fine.

Comment (encukou) [reply CHANGES]: Fixing in https://github.com/python-ldap/python-ldap/pull/73

Comment (tiran) [reply CHANGES]: thanks, I opened #76 to track the issue.

**Pull request merged**